mysite/static/js/importer.js

```diff
@@ -182,7 +179,6 @@ Preparation = {
         fireunit.ok(typeof query != 'undefined', "query: " + query);
 
         var data = {'format': 'success_code'};
-
```

Comment:
@onceuponatimeforever I'm wondering if enablethrobbers... and findThrobbersForDia blocks above can be removed...